com.github.Anuken.Mindustry.yml

```diff
@@ -32,7 +32,6 @@ modules:
       - type: git
         url: 'https://github.com/Anuken/Mindustry'
         tag: 'v135'
-        commit: c24c5c6196cd89bdc17a079a2a8acd372cc260d9
```

Comment:
Add this again, for reproducibility.

Reply:
Didn't you just tell me to remove it?

Reply:
No, it's just GitHub being GitHub. We can see in the diff https://github.com/flathub/com.github.Anuken.Mindustry/pull/39#discussion_r807190813 that I re-add it. That's just how GitHub works.